src/Jackett.Common/Indexers/SubsPlease.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    [ExcludeFromCodeCoverage]
+    public class SubsPlease : BaseWebIndexer
+    {
+        public SubsPlease(IIndexerConfigurationService configService, Utils.Clients.WebClient wc, Logger l, IProtectionService ps, ICacheService cs)
+            : base(id: "SubsPlease",
+                   name: "SubsPlease",
+                   description: "SubsPlease - A better HorribleSubs/Erai replacement",
+                   link: "https://subsplease.org/",
+                   caps: new TorznabCapabilities
+                   {
+                       TvSearchParams = new List<TvSearchParam>
+                       {
+                           TvSearchParam.Q, TvSearchParam.Season, TvSearchParam.Ep
+                       }
+                   },
+                   configService: configService,
+                   client: wc,
+                   logger: l,
+                   p: ps,
+                   cacheService: cs,
+                   configData: new ConfigurationData())
+        {
+            Encoding = Encoding.UTF8;
+            Language = "en-us";
+            Type = "public";
+
+            // Configure the category mappings
+            AddCategoryMapping(1, TorznabCatType.TVAnime, "Anime");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+            var releases = await PerformQuery(new TorznabQuery());
+
+            await ConfigureIfOK(string.Empty, releases.Any(), () =>
+                throw new Exception("Could not find releases from this URL"));
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        // If the search string is empty use the latest releases
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+            => query.IsTest || string.IsNullOrWhiteSpace(query.SearchTerm)
+            ? await FetchNewReleases()
+            : await PerformSearch(query);
+
+        private async Task<IEnumerable<ReleaseInfo>> PerformSearch(TorznabQuery query)
+        {
+            var queryParameters = new NameValueCollection
+            {
+                { "f", "search" },
+                { "tz", "America/New_York" },
+                { "s", query.SearchTerm }
+            };
+            var response = await RequestWithCookiesAndRetryAsync(SiteLink + "/api/?" + queryParameters.GetQueryString());
```

Comment:
Add a global constant with the API endpoint
```
 private string ÀpiUrl => SiteLink + "/api/?";
```

https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Indexers/AwesomeHD.cs#L22

Reply:
Fixed.